### PR TITLE
Adds version number

### LINKED ISSUE #442: Add version property to pdc_describe form
This will be on the curator only tab.  This can only be whole numbers

Acceptance criteria
- [x] it should not be user editable
- [x] it should be curator editable 
- [x] it should appear in the DataCite record 
- [x] it should be set by default to 1

### PR DESCRIPTION
**Pull request by hectorcorrea**
Closes #442

### CODE REVIEW

File: app/views/works/_curator_controlled_form.html.erb [hunk 1]
```diff
@@ -1,18 +1,29 @@
...
   <!-- DOI field (mostly for legacy DSpace works) -->
   <div class="field">
-    DOI <span class="text-muted field-hint">(enter the Digital Object Identifier, e.g. 10.80021/62qy-nj36)</span><br>
     <% if  @wizard_mode || @work.persisted? %>
-       <%= @work.doi %>
+      DOI: <br>
+      <%= @work.doi %>
     <% else %>
-       <input type="text" id="doi" name="doi" class="input-text-long" value="<%= @work.doi %>"/>
+      DOI: <span class="text-muted field-hint">(enter the Digital Object Identifier, e.g. 10.80021/62qy-nj36)</span><br>
```
Comment (carolyncole) [inline app/views/works/_curator_controlled_form.html.erb:8]: 🎉

**Review by carolyncole — CHANGES_REQUESTED** [review]
It would be nice to have a test to make sure the version is showing up in the datacite xml after it is set in the UI, or at the model level.

**Comment by hectorcorrea** [comment]
@carolyncole this test covers the Datacite XML generation: 

https://github.com/pulibrary/pdc_describe/pull/457/files#diff-97a638a710a9092438e481d2f3458a2059e94b561474464692112cc77a1a7064R26

**Comment by carolyncole** [comment]
@hectorcorrea Totally missed that.  Thank you!

**Review by carolyncole — APPROVED** [review]

**Pull request merged**